Dockerfile

```
@@ -0,0 +1,38 @@
+FROM python:3.12.0a7-buster as base
+
+# Install poetry
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH /root/.local/bin:$PATH
+
+# Make workspace
+WORKDIR /app
+COPY poetry.toml .
+COPY pyproject.toml .
```

We can combine these in a single line, if we prefer it:
```suggestion
COPY poetry.toml pyproject.toml ./
```